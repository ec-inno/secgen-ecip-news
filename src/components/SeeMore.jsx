import React from 'react';
import PropTypes from 'prop-types';

const SeeMore = ({ label, ariaLabel, onClick }) => (
  <div className="ecl-row ecl-u-mt-l">
    <div className="ecl-col-sm-12 ecl-col-md-12">
      <nav className="ecl-pagination" aria-label="Pagination">
        <ul className="ecl-pagination__list">
          <li className="ecl-pagination__item ecl-pagination__item--next">
            <a
              onClick={onClick}
              aria-label={ariaLabel}
              href="#"
              className="ecl-pagination__link ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon-after"
            >
              <span className="ecl-link__label">{label}</span>
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
);

SeeMore.propTypes = {
  ariaLabel: PropTypes.string,
  label: PropTypes.string,
  onClick: PropTypes.func,
};

SeeMore.defaultProps = {
  ariaLabel: '',
  label: '',
  onClick: () => {},
};

export default SeeMore;
