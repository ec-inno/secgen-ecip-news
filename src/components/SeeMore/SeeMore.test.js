import React from 'react';
import renderer from 'react-test-renderer';
import SeeMore from './SeeMore';

describe('SeeMore', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SeeMore />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
