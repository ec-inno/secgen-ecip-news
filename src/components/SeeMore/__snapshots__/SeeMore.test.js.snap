// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeeMore renders correctly 1`] = `
<div
  className="ecl-row ecl-u-mt-l"
>
  <div
    className="ecl-col-sm-12 ecl-col-md-12"
  >
    <nav
      aria-label="Pagination"
      className="ecl-pagination"
    >
      <ul
        className="ecl-pagination__list"
      >
        <li
          className="ecl-pagination__item ecl-pagination__item--next"
        >
          <a
            aria-label=""
            className="ecl-pagination__link ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon-after"
            href="#"
            onClick={[Function]}
          >
            <span
              className="ecl-link__label"
            >
              
            </span>
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
